### PR TITLE
US975260: Refactor to remove HPE references in package names

### PR DESCRIPTION
**Pull request by raghavendran-gopalakrishnan**
https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=975260

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -77,44 +77,56 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>com.fasterxml.jackson.core</groupId>
+                <artifactId>jackson-annotations</artifactId>
+                <version>2.18.1</version>
+                <scope>runtime</scope>
```
Comment (dermot-hardy) [inline pom.xml]: We don't put scopes in dependency management.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: worker-taskstowing/pom.xml [hunk 2]
```diff
@@ -32,6 +32,11 @@
     </parent>
 
     <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <scope>runtime</scope>
+        </dependency>
```
Comment (dermot-hardy) [inline worker-taskstowing/pom.xml]: Is this dependency actually required?  It's very unusual to have runtime-scope annotation dependencies.

Comment (raghavgopal) [reply worker-taskstowing/pom.xml]: It was being looked for, while worker container starts in integration test.

Comment (dermot-hardy) [reply worker-taskstowing/pom.xml]: What is the exception?

Comment (raghavgopal) [reply worker-taskstowing/pom.xml]: https://sou-jenkins2.swinfra.net/job/JobService/job/JobService~worker-taskstowing~Test~CI/1/console

> 2024-11-19T18:29:39.883512340Z [INFO] DOCKER> [localhost:49227/sepg-dockerhub-image/jobservice-9034-worker-taskstowing:1.0.0-Test-SNAPSHOT] "worker-taskstowing": Start container 8ea67304fae4
2024-11-19T18:29:40.311794792Z [INFO] DOCKER> [localhost:49227/sepg-dockerhub-image/jobservice-9034-worker-taskstowing:1.0.0-Test-SNAPSHOT] "worker-taskstowing": Waiting on url http://16.32.3.13:49233/healthcheck with method HEAD for status 200..399.
2024-11-19T18:29:40.395643924Z 18:29:40.393 worker-taskstowing> [2024-11-19 18:29:40.391Z #007.??? INFO  -            -   ] startup.sh: Running startup scripts...
2024-11-19T18:29:40.404170825Z 18:29:40.402 worker-taskstowing> [2024-11-19 18:29:40.400Z #007.??? INFO  -            -   ] startup.sh: Running install-ca-cert-base.sh...
2024-11-19T18:29:40.429534431Z 18:29:40.427 worker-taskstowing> [2024-11-19 18:29:40.423Z #00E.??? INFO  -            -   ] install-ca-cert-base.sh: Not installing CA Certificate.
2024-11-19T18:29:40.434053615Z 18:29:40.432 worker-taskstowing> [2024-11-19 18:29:40.431Z #007.??? INFO  -            -   ] startup.sh: Running install-ca-cert-java.sh...
2024-11-19T18:29:40.450422133Z 18:29:40.448 worker-taskstowing> [2024-11-19 18:29:40.446Z #016.??? INFO  -            -   ] install-ca-cert-java.sh: Not installing CA Certificate for Java
2024-11-19T18:29:40.454047914Z 18:29:40.452 worker-taskstowing> [2024-11-19 18:29:40.451Z #007.??? INFO  -            -   ] startup.sh: Startup scripts completed
2024-11-19T18:29:40.457068341Z 18:29:40.455 worker-taskstowing> [2024-11-19 18:29:40.454Z #007.??? INFO  -            -   ] startup.sh: The RUNAS_USER environment variable is not set, subsequent commands will be run as the default user.
2024-11-19T18:29:41.620598142Z 18:29:41.618 worker-taskstowing> [18:29:41.578Z #a37.001 INFO  -            -   ] o.h.v.internal.util.Version: HV000001: Hibernate Validator 8.0.1.Final
**2024-11-19T18:29:42.153639631Z 18:29:42.152 worker-taskstowing> java.lang.NoClassDefFoundError: com/fasterxml/jackson/annotation/JsonTypeInfo$Value**
2024-11-19T18:29:42.153875396Z 18:29:42.152 worker-taskstowing> 	at com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector.findPolymorphicTypeInfo(JacksonAnnotationIntrospector.java:670)
2024-11-19T18:29:42.154030015Z 18:29:42.152 worker-taskstowing> 	at com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector._findTypeResolver(JacksonAnnotationIntrospector.java:1579)
2024-11-19T18:29:42.154038789Z 18:29:42.152 worker-taskstowing> 	at com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector.findTypeResolver(JacksonAnnotationIntrospector.java:677)

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**